openhtf/plugs/acroname/power_module_plug.py

```diff
@@ -0,0 +1,309 @@
+# Copyright 2014 Google Inc. All Rights Reserved.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+"""Acroname power module plug for OpenHTF."""
+
+
+import logging
+import time
+from time import sleep
+
+import openhtf.conf as conf
+import openhtf.plugs as plugs
+
+from brainstem import discover
+from brainstem.link import Spec
+from brainstem.stem import MTMPM1
+from brainstem.stem import MTMEtherStem
+from brainstem.stem import USBStem
+from brainstem.defs import model_info
+
+# _LOG = logging.getLogger("openhtf.example.plugs.acroname.power_module_plug.py")
+_LOG = logging.getLogger(__name__)
+
+conf.Declare('serial_number_of_EtherStem',
```

Comment:
These config keys should be nested under something like EtherStem, perhaps like:
```
etherstem:
    serial_number: <number>
    pm_default_voltage: <voltage>
    pm_default_current_limit: <current>
```